addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4853,6 +4853,14 @@ registry.SnippetSave = SnippetOptionWidget.extend({
                                 const snippetName = dialog.el.querySelector('.o_we_snippet_name_input').value;
                                 const targetCopyEl = this.$target[0].cloneNode(true);
                                 delete targetCopyEl.dataset.name;
+                                // Removing the ids is especially important for snippets such as
+                                // tabs snippet which uses anchors. Indeed, it is important that 
+                                // the anchors of the saved tabs located on the editor and the
+                                // anchors of the tabs on the webpage do not refer to the same
+                                // elements.
+                                for (const idTargetCopyEl of targetCopyEl.querySelectorAll('[id]')) {
+                                    idTargetCopyEl.id = "";
+                                }
```

Comment:
@loco-odoo After some more thinking, I think we should not do this like that. Could you change this PR to:

- Remove the code I am commenting here, to not remove the `id` of saved snippets: we could want to keep IDs. For example, design a nice pricing snippet, set an ID to it via the "anchor" button, name it "pricing" so that you can make links to that pricing snippets, save the snippet -> you want the ID to be kept so you can drop your "pricing" snippet everywhere and make links to it easily. More than that, we don't know what users in stable might want to do: best focused on fixing this very specific bug only.
- The bug is actually not there anymore in 16.0 since the editor is outside of the iframe of the page. The ID of the website block and the ID of the snippet definition in the right panel are thus not conflicting with each other. This PR can thus be forward-port up to 16 not included. You can change your comment "remove me in master" to "remove me in 16.0"
- Limit the ID removal in `_computeSnippetTemplates` to tab pane IDs like I said earlier. Same reason as explained above: we only want to fix this tab behavior without risking removing IDs users may want to keep.
- Adapt your commit message accordingly if needed.